components/datetime/src/options/mod.rs

```diff
@@ -5,69 +5,69 @@
 //! `DateTimeFormatOptions` is a bag of options which, together with `LanguageIdentifier`,
 //! define how dates will be formatted be a `DateTimeFormat` instance.
 //!
-//! Each variant of the bag is a combination of settings definiting how to format
+//! Each variant of the bag is a combination of settings defining how to format
 //! the date, with an optional `Preferences` which represent user preferences and
 //! may alter how the selected pattern is formatted.
 //!
 //! # Examples
 //!
 //! ```
-//! use icu_datetime::{DateTimeFormatOptions, options::style};
+//! use icu_datetime::{DateTimeFormatOptions, options::length};
 //!
-//! let options = DateTimeFormatOptions::Style(
-//!     style::Bag {
-//!          date: Some(style::Date::Medium),
-//!          time: Some(style::Time::Short),
+//! let options = DateTimeFormatOptions::Length(
+//!     length::Bag {
+//!          date: Some(length::Date::Medium),
+//!          time: Some(length::Time::Short),
 //!         ..Default::default()
 //!     }
 //! );
 //! ```
 //!
-//! At the moment only the `Style` bag works, and we plan to extend that to support
+//! At the moment only the `Length` bag works, and we plan to extend that to support
```

Comment:
@zbraniecki is this still true or should I remove this future looking comment?

Reply:
We started adding support for components, but I don't think we can claim ECMA-402 support yet. @gregtatum is driving that.